db/migrate/20190901111825_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :role, :integer
```

Comment:
I would prefer not to use magic numbers here, how about string? https://sipsandbits.com/2018/04/30/using-database-native-enums-with-rails/#integercolumnsarehardtounderstandwithoutactualcontext and also follow down for database integrity too :)

Reply:
I've added new issue to add PostgreSQL (#46)